src/analyses/controllers.clj

```diff
@@ -6,16 +6,19 @@
 
 (defn add-quicklaunch
   [user quicklaunch]
-  (clients/validate-submission
-   {:quicklaunch quicklaunch
-    :app         (clients/get-app user system-id (:app_id quicklaunch))
-    :system-id   system-id
-    :user        user})
-  (persist/add-quicklaunch user quicklaunch))
+  (let [{:keys [version_id] :as app} (clients/get-app user system-id (:app_id quicklaunch))
+        quicklaunch                  (assoc quicklaunch :app_version_id version_id)]
```

Comment:
This is more of a question than a comment. Is the purpose of this PR simply to get the service working with the updated schema?

My concern was that if the app version isn't included in the quick launch definition request then the parameter IDs will probably change between app versions. I suspect that this concern won't apply in the final version of the code, however.

Reply:
That's correct on both counts.

This round of PRs is only to get the services compatible with the schema changes in cyverse-de/de-database#10, without new requirements in endpoint requests or in the UI. This usually means using the latest version of an app in the services for now.

I agree that eventually this endpoint will need to require the app version ID in the request.

Reply:
I thought that was probably the case. Thanks for the confirmation.